content/docs/command-reference/stage/add.md

```diff
@@ -0,0 +1,432 @@
+  stage. This option behaves like `-o` but registers the file in a `metrics`
+  field inside the `dvc.yaml` stage. Metrics are usually small, human readable
+  files (JSON or YAML) with scalar numbers or other simple information that
+  describes a model (or any other data artifact). See `dvc metrics` to learn
+  more about _metrics_.
+
+- `-M <path>`, `--metrics-no-cache <path>` - the same as `-m` except that DVC
+  does not track the metrics file (same as with `-O` above). This means that
+  they are never cached, so it's up to the user to manage them separately. This
+  is typically desirable with _metrics_ because they are small enough to be
+  tracked with Git directly.
+
+- `--plots <path>` - specify a plot metrics file produces by this stage. This
+  option behaves like `-o` but registers the file in a `plots` field inside the
+  `dvc.yaml` stage. Plot metrics are data series stored in tabular (CSV or TSV)
+  or hierarchical (JSON or YAML) files, with complex information that describes
+  a model (or any other data artifact). See `dvc plots` to learn more about
+  plots.
+
+- `--plots-no-cache <path>` - the same as `--plots` except that DVC does not
+  track the plots file (same as with `-O` and `-M` above). This may be desirable
+  with _plots_, if they are small enough to be tracked with Git directly.
+
+- `-w <path>`, `--wdir <path>` - specifies a working directory for the `command`
+  to run in (uses the `wdir` field in `dvc.yaml`). Dependency and output files
+  (including metrics and plots) should be specified relative to this directory.
+  It's used by `dvc repro` to change the working directory before executing the
+  `command`.
+
+- `-f`, `--force` - overwrite an existing stage in `dvc.yaml` file without
+  asking for confirmation.
+
+- `--always-changed` - always consider this stage as changed (uses the
+  `always_changed` field in `dvc.yaml`). As a result `dvc status` will report it
+  as `always changed` and `dvc repro` will always execute it.
+
+  > Note that regular `.dvc` files (without dependencies) are automatically
+  > considered "always changed", so this option has no effect in those cases.
+
+- `--external` - allow writing outputs outside of the DVC repository. See
+  [Managing External Data](/doc/user-guide/managing-external-data).
+
+- `--desc <text>` - user description of the stage (optional). This doesn't  
+  affect any DVC operations.
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+Let's create a <abbr>DVC project</abbr> and a stage (that counts the number of
+lines in a `test.txt` file):
+
+```dvc
+$ mkdir example && cd example
+$ git init
+$ dvc init
+$ mkdir data
+$ dvc stage add -n count \
+          -d test.txt \
+          -o lines \
+          "cat test.txt | wc -l > lines"
```

Comment:
I assume there are no changes to the examples so won't review deeply. But 💅 can we please re-indent the code samples? E.g.:
```suggestion
$ dvc stage add -n count \
                -d test.txt \
                -o lines \
                "cat test.txt | wc -l > lines"
```